### PR TITLE
Primata/314 weth eth amount type

### PR DESCRIPTION
**Pull request by 0xPrimata**
# Summary
- **RFCs**: https://github.com/movementlabsxyz/movement/issues/405
- **Categories**: `protocol-units`

Adds bridge_transfers(bridgeTransferId) view function to move contracts

# Changelog

# Testing

Testing seems to cover but there is one issue with view functions behavior. it's altering the state and we would expect it to not do so.

# Outstanding issues

### CODE REVIEW

**Comment by andygolay** [comment]
@0xPrimata any idea why the `[suzuka-full-node-local (ubuntu-22.04, x86_64, buildjet-16vcpu-ubuntu-2204)]` job is failing?

**Review by andygolay — CHANGES_REQUESTED** [review]
gm @0xPrimata 

- Any idea why the `[suzuka-full-node-local (ubuntu-22.04, x86_64, buildjet-16vcpu-ubuntu-2204)]` job is failing?

- Is `test_swap` the integration test for these changes?

**Review by andygolay — COMMENTED** [review]
Tests pass and make sense; pending CI passing, LGTM.

Will wait on CI resolution before approving.

**Comment by andygolay** [comment]
> gm @0xPrimata
> 
> * Any idea why the `[suzuka-full-node-local (ubuntu-22.04, x86_64, buildjet-16vcpu-ubuntu-2204)]` job is failing?
> * Is `test_swap` the integration test for these changes?

The suzuka test passes now after re-running CI. 

LGTM

File: protocol-units/bridge/integration-tests/tests/eth_movement.rs
Comment (0xmovses) [inline protocol-units/bridge/integration-tests/tests/eth_movement.rs:224]: 👍

File: protocol-units/bridge/shared/src/bridge_contracts.rs
Comment (0xmovses) [inline protocol-units/bridge/shared/src/bridge_contracts.rs:62]: 👍

**Review by 0xmovses — APPROVED** [review]
This looks good! :shipit:

**Pull request merged**
